app/server.js

```diff
@@ -0,0 +1,65 @@
+const ttn = require("ttn")
+const app = require("express")()
+const http = require("http").Server(app)
+const bodyParser = require("body-parser")
+
+const googleMapsClient = require("@google/maps").createClient({
+  key: "xxxxxxxxxxxxxxxxxxxxxxxxxx"
+})
+
+var appID = "xxxxxxxxxxxxxxxxx"
+var accessKey = "ttn-account-v2.xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+var macAddress1 = ""
+var macAddress2 = ""
+var macAddress3 = ""
+
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: false }))
+
+//this is the default home page of our application
+app.get("/", function(req, res){
+  res.sendFile(__dirname + "/map.html")
+})
+
+app.get("/generator", function(req, res){
+  googleMapsClient.geolocate({
+    "considerIp": false,
+    "wifiAccessPoints": [
+      {
+        "macAddress": macAddress1
+      },
+      {
+        "macAddress": macAddress2
+      },
+      {
+        "macAddress": macAddress3
+      }
+    ]
+  }, function(err, response) {
+    console.log(response)
+    console.log(response.json.location)
+    return res.json(response.json.location)
+  })
+})
+
+//opening mqtt connection with ttn
+ttn.data(appID, accessKey)
+.then(function(client) {
```

Comment:
This indentation doesn't seem right. Please run a code formatter and configure your editor to format on save. I.e. something like https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode if you use VS Code